pydicom/sopclass_uids.py

```diff
@@ -0,0 +1,299 @@
+"""
+Storage SOP Class UIDs auto-generated by generate_storage_sopclass_uids.py
+"""
```

Comment:
I wasn't sure about the name  (didn't  want to have it too long), but you are right - will change it.
About the extra folder - I think it would be OK, but it would break downwards compatibility,  so I leave that to @darcymason to decide.